Prox/Prox/PlaceCarousel/PlaceCarouselViewController.swift

```diff
@@ -189,22 +196,20 @@ class PlaceCarouselViewController: UIViewController {
 
         // apply the constraints
         NSLayoutConstraint.activate(constraints, translatesAutoresizingMaskIntoConstraints: false)
+
+        toggleLoadingUI(loading: true)
+
+        if let location = self.locationMonitor.getCurrentLocation() {
+            updatePlaces(forLocation: location)
+        }
     }
 
     override func viewWillAppear(_ animated: Bool) {
         super.viewWillAppear(animated)
 
-        guard let location = locationMonitor.getCurrentLocation() else {
-            return
+        if shouldStartMonitoringLocation() {
```

Comment:
#218 is addressing the place fetching on location update issues. I don't want competing implementations causing problems on merge